src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The bulldozer has just started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("The bulldozer was stopped");

```

Comment:
Similar to the 'doWork' method, the stop message should be more informative. Include the type of machine (Bulldozer) to clarify which machine has stopped working.